[2023-06-21T03:00:00.914+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: wrapper_ex_dag.execute_function_task scheduled__2023-06-21T02:00:00+00:00 [queued]>
[2023-06-21T03:00:00.918+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: wrapper_ex_dag.execute_function_task scheduled__2023-06-21T02:00:00+00:00 [queued]>
[2023-06-21T03:00:00.918+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-21T03:00:00.924+0000] {taskinstance.py:1327} INFO - Executing <Task(_PythonDecoratedOperator): execute_function_task> on 2023-06-21 02:00:00+00:00
[2023-06-21T03:00:00.927+0000] {standard_task_runner.py:57} INFO - Started process 960 to run task
[2023-06-21T03:00:00.929+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'wrapper_ex_dag', 'execute_function_task', 'scheduled__2023-06-21T02:00:00+00:00', '--job-id', '431', '--raw', '--subdir', 'DAGS_FOLDER/Python-Delta-Lake/Dags/WrapperDag.py', '--cfg-path', '/tmp/tmp67_84sym']
[2023-06-21T03:00:00.930+0000] {standard_task_runner.py:85} INFO - Job 431: Subtask execute_function_task
[2023-06-21T03:00:00.954+0000] {task_command.py:410} INFO - Running <TaskInstance: wrapper_ex_dag.execute_function_task scheduled__2023-06-21T02:00:00+00:00 [running]> on host 68331b7f6fd1
[2023-06-21T03:00:00.991+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='phomsophaN@saccounty.net' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='wrapper_ex_dag' AIRFLOW_CTX_TASK_ID='execute_function_task' AIRFLOW_CTX_EXECUTION_DATE='2023-06-21T02:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-21T02:00:00+00:00'
[2023-06-21T03:00:00.991+0000] {CsvDagWrapper.py:22} INFO - CSV Delta Class Initialted
[2023-06-21T03:00:00.991+0000] {logging_mixin.py:149} INFO - Environment successfully parsed! Environment is Local
[2023-06-21T03:00:00.991+0000] {CsvDagWrapper.py:39} INFO - Environment successfully parsed! Environment is Local
[2023-06-21T03:00:00.991+0000] {logging_mixin.py:149} INFO - Table/s successfully parsed! Table:  ['CASE', 'PTR']
[2023-06-21T03:00:00.991+0000] {CsvDagWrapper.py:44} INFO - Table/s successfully parsed! Table:  ['CASE', 'PTR']
[2023-06-21T03:00:00.992+0000] {logging_mixin.py:149} INFO - Number of tables to be processed = 2
[2023-06-21T03:00:00.992+0000] {logging_mixin.py:149} INFO - TABLE_NAME IN ('CASE', 'PTR')
[2023-06-21T03:00:00.992+0000] {CsvDagWrapper.py:60} INFO - Works so far....
[2023-06-21T03:00:03.384+0000] {CsvDagWrapper.py:63} INFO - test
[2023-06-21T03:00:33.414+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/decorators/base.py", line 220, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/Python-Delta-Lake/Dags/WrapperDag.py", line 46, in execute_function_task
    task.RunCsvToDelta("Local",["CASE","PTR"],False)
  File "/opt/airflow/dags/Python-Delta-Lake/Csv2Delta/CsvDagWrapper.py", line 71, in RunCsvToDelta
    sql_connection_1 = pyodbc.connect(sql_server_connection_string)
pyodbc.OperationalError: ('HYT00', '[HYT00] [Microsoft][ODBC Driver 18 for SQL Server]Login timeout expired (0) (SQLDriverConnect)')
[2023-06-21T03:00:33.420+0000] {taskinstance.py:1345} INFO - Marking task as FAILED. dag_id=wrapper_ex_dag, task_id=execute_function_task, execution_date=20230621T020000, start_date=20230621T030000, end_date=20230621T030033
[2023-06-21T03:00:33.432+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 431 for task execute_function_task (('HYT00', '[HYT00] [Microsoft][ODBC Driver 18 for SQL Server]Login timeout expired (0) (SQLDriverConnect)'); 960)
[2023-06-21T03:00:33.475+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-06-21T03:00:33.483+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
